doc/reference/reference_lua/key_def.rst

```diff
@@ -19,17 +19,18 @@ to extract or compare the index key values.
 
     :param table parts: field numbers and types.
                         There must be at least one part and it
-                        must have at least fieldno and type.
+                        must have at least ``fieldno`` and ``type``.
```

Comment:
@Totktonada 
Must all parts have the ``fieldno`` and ``type`` attributes? Or can there be only one part with those attributes, while the other parts don't have those attributes?

Reply:
All parts must have `fieldno` or `field` attribute (see #962). All parts must have `type` attribute. Other attributes are optional. The `parts` array should have at least one item.